### PR TITLE
add panstarrs entry

### PR DESCRIPTION
**Pull request by ttdu**

### CODE REVIEW

**Comment by ttdu** [comment]
Paging @rlwastro for review. I want to confirm:

- [x] This is an adequate summary of PS1. AWS listings are generally 2-3 sentences (see e.g. https://registry.opendata.aws/hst/) so we don't need to be too verbose here.
- [x] The licensing is weird to me. I guess as a member of the PS1 consortium we can release our share of the data to the public as we see fit? Feel free to correct this as needed
- [x] My inclination is to link to a NB showing how to use PS1 data, even if it doesn't use AWS, since it's better than nothing. At a later date, we can work on writing a NB that takes advantage of the cloud to do something interesting

**Comment by ttdu** [comment]
Tagging @sshen-stsci for visibility

File: datasets/panstarrs.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Name: Pan-STARRS PS1 Survey
+Description: |
+  The PS1 surveys used a 1.8 meter telescope and its 1.4 Gigapixel camera to image the sky in five broadband filters. The largest of these surveys provides coverage of the entire sky north of -30 degrees declination.
```
Comment (rlwastro) [inline datasets/panstarrs.yaml]:
```suggestion
  The PS1 surveys used a 1.8 meter telescope and its 1.4 Gigapixel camera to image the sky in five broadband filters. The largest of these surveys provides coverage of the entire sky north of -30 degrees declination, with approximately 10 observation epochs across 3 years in each filter.
```

File: datasets/panstarrs.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+Documentation: https://outerspace.stsci.edu/display/PANSTARRS/
+Contact: archive@stsci.edu
+ManagedBy: "[Space Telescope Science Institute](http://www.stsci.edu/)"
+Citation: Please see the documentation for full citation instructions.
+UpdateFrequency: Never
+Tags:
+  - astronomy
+License: STScI hereby grants the non-exclusive, royalty-free, non-transferable, worldwide right and license to use, reproduce, and publicly display in all media data from the PS1 surveys.
```
Comment (rlwastro) [inline datasets/panstarrs.yaml:11]: The license stuff is weird to me too.  Back in 2017 we made this statement about the copyright:

> These images are copyrighted by the University of Hawaii and are free for non-commercial use.  Hawaii is currently discussing the precise copyright statement for commercial use.  I believe that the images will be free for all to use but don't know that for certain yet.  We will be adding a statement to our site once that is settled.

We never did get a response from Hawaii on the details of their preferred license.  But they have approved the move of the data to the AWS public data, so the license needs to be whatever is appropriate for AWS.  We could try again to get a definitive answer from Hawaii on this, or we can take their approval of the AWS data availability as also approving the necessary license.

File: datasets/panstarrs.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+Resources:
+  - Description: PS1 DR1 and DR2 files
```
Comment (rlwastro) [inline datasets/panstarrs.yaml]: They are actually image files (and specifically FITS files).  But in the future we could add some other PS1 data such as the bulk catalog csv files.  I'll leave it up to you whether we should be more specific about the data description.

File: datasets/panstarrs.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    ARN: arn:aws:s3:::stpubdata/ps1
+    Region: us-east-1
+    Type: S3 Bucket
+    RequesterPays: False
+DataAtWork:
+  Tutorials:
+  - Title: Light Curves with PanSTARRS
+    URL: https://spacetelescope.github.io/mast_notebooks/notebooks/PanSTARRS/PS1_DR2_TAP/PS1_DR2_TAP.html
```
Comment (rlwastro) [inline datasets/panstarrs.yaml]: As you said, this notebook does not use the S3 data (or even the images) at all, so it is marginally relevant.  Using this link as a placeholder is OK with me though.

**Review by rlwastro — APPROVED** [review]

File: datasets/panstarrs.yaml (hunk 2 above)
Comment (ttdu) [reply datasets/panstarrs.yaml:11]: Uh, ok. I guess if Hawaii were concerned about this, they would have followed up. Right now, the data is available in AWS with no license listed, so let's go for option 2.

File: datasets/panstarrs.yaml (hunk 3 above)
Comment (ttdu) [reply datasets/panstarrs.yaml]: Oops, yes -- I will specify that we are including image files.

**Pull request merged**